config.py

```diff
@@ -39,11 +39,20 @@ class CIConfig(Config):
 
 
 class DevConfig(Config):
+    PORT = 8003
```

Comment:
Perhaps the base config can also set the PORT to some default value (80 / 8003).

Reply:
I don't think it will be used in the base config as it will be kicked off by the Procfile?

Reply:
True enough. It was more a suggestion to avoid a KeyError when testing with `APP_CONFIG=Config python run.py`

Reply:
Done